src/router/routes.ts

```diff
@@ -44,19 +54,25 @@ export const routes = [
     }
   },
   {
-    path: '/limitedRestorationToFull',
-    name: RouteNames.RESTORATION_CONVERSION,
-    component: Restoration,
+    path: '/limitedRestorationExtension',
+    name: RouteNames.RESTORATION_EXTENSION,
+    component: LimitedRestorationExtension,
+    props: route => ({
+      restorationId: route.query['restoration-id']
```

Comment:
👍 

I think I remember a refactoring ticket talking about this. Please add a comment to that ticket indicating what you have already done and maybe pointing to this code as an example for other filings.